signer-npm/src/lib.rs

```diff
@@ -223,13 +228,19 @@ pub fn transaction_sign(
         .into_serde()
         .map_err(|e| JsValue::from(format!("Error parsing parameters: {}", e)))?;
 
+    let msg = match unsigned_message {
+        MessageTxAPI::Message(m) => m,
+        MessageTxAPI::SignedMessage(_) => {
+            return Err(JsValue::from("Can't serialize SignedMessage"))
```

Comment:
can this error message be made clearer? the function is called "transaction_sign" so this error doesn't seem to bring clarity of why the function errored, perhaps "Attempted to sign a SignedMessage" would be more explicit